common.gradle

```diff
@@ -16,6 +16,9 @@ repositories {
     maven {
         url "http://nifty-gui.sourceforge.net/nifty-maven-repo"
     }
+    maven {
```

Comment:
Perhaps this should go into the jme3-lwjgl3 project instead, since no code outside of it should be using LWJGL 3 anyway.